partner_priority/data/partner_priority_data.xml

```diff
@@ -0,0 +1,27 @@
+<!-- Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>

```

Comment:
```suggestion
<odoo noupdate="1">
```

And remove `<data noupdate="1">`

Reply:
The reason I have the noupdate="1" is I think it's best practice to not re-import these when the module is updated, because if the company/user using this changes the values for their own use it shouldn't be changed back if its updated. I'm open to discussion though, I almost just excluded this data file altogether as it's not necessarily needed. Thoughts?

Reply:
@patrickrwilson I think you misread what @nikul-serpentcs is proposing. The attribute 'noupdate' can now be part of the odoo element. The data element is deprecated. So you can (and should) still use noupdate, but move it to the odoo element.

Reply:
Yes, @NL66278 you are correct, I didn't realize you can put the 'noupdate' in the odoo element so I was misreading @nikul-serpentcs comment. Thanks. 

I've made this adjustment and squashed commits.